zerberus/zone.cpp

```diff
@@ -42,13 +42,13 @@ Zone::~Zone()
 //   match
 //---------------------------------------------------------
 
-bool Zone::match(Channel* c, int k, int v, Trigger et, double rand)
+bool Zone::match(Channel* c, int k, int v, Trigger et, double rand, int cc, int ccVal)
       {
 
-      if ((k >= keyLo)
-         && (k <= keyHi)
-         && (v >= veloLo)
-         && (v <= veloHi)
+      if ((k >= keyLo || et == Trigger::CC)
```

Comment:
I know it was already in place in the Trigger enum but does it really make sense to use `Trigger::CC` here? It doesn't map the trigger value in the spec. Potentially, it could be use for start_locc, start_hicc, stop_locc, stop_hicc but here, can't we just use the values of cc or ccVal?